packages/vitepress-plugin-git-changelog/src/client/components/Contributors.vue

```diff
@@ -11,6 +12,8 @@ const options = defu(inject(InjectionKey, {}), defaultOptions)
 const { t } = useI18n()
 const { authors, useHmr } = useChangelog()
 
+const { localeIndex } = useData()
```

Comment:
May I ask what is `localeIndex`? VitePress doesn't have much documentations about this field, and I not quite sure what's the difference between `lang` and `localeIndex` after looking at the source code of VitePress with the following Pull Requests:

- https://github.com/vuejs/vitepress/pull/3862/files
- https://github.com/vuejs/vitepress/pull/1339/files#diff-5a628d1968485980dfc86e06a4a9ed3ed19a912261045f65d3e343e57ad91a97R14

Reply:
I'm sorry, but i cant understand this too)
This variables has the same values in live usage. 
The choice was only due to the fact that it was the one I had seen used elsewhere.

Reply:
maybe lang can be more useful in i18n

`lang` use only lang codes
`localeIndex` can show "root" for any locale

Reply:
1. Perhaps we could file an issue to VitePress to request more explanation on this field (and prevent anti-patterns and bad practices).
2. Besides all of that, VitePress doesn't have much documentations about `root` too... Currently, all the Nolebase Integrations are using lang or helper-calculated `lang` from paths and urls. Do you have any of `root` used to determine and configure for locales that I could reference to for better understanding?

Much appriciated.

cc @northword

Reply:
I'm sorry, I really don't know much about the i18n related parts of VitePress, so I can't give much advice.

From the information mentioned so far, both `lang` and `langIndex` seem to make the feature work?

If it is not necessary to use `langIndex` to work properly, I think `lang` is a better choice to keep the Nolebase plugin options consistent.